TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/DeviceConnection.java

```diff
@@ -0,0 +1,29 @@
+package com.novoda.tpbot.bot;
+
+abstract class DeviceConnection {
```

Comment:
Needed an abstraction here to force clients to pass the `DeviceConnectionListener`. Client's are forced to have these methods as a minimum because we can expect a `connect`, `disconnect` and `send` event.